tests/sample/ext/drv/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ target_include_directories("sample_ebpf_ext" PRIVATE
   "${CMAKE_SOURCE_DIR}/include/kernel"
   "${CMAKE_SOURCE_DIR}/libs/platform"
   "${CMAKE_SOURCE_DIR}/libs/platform/kernel"
+  "${CMAKE_SOURCE_DIR}/libs/store_helper/kernel"
```

Comment:
why do we need to add `libs/store_helper/kernel` to include path?

Reply:
This can be safely removed.